DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
From what I read, `std::unordered_map` is even worse than `std::map` in terms of memory usage but provides faster insertion and retrieval time.